diff --git a/.next/node_modules/esbuild-caa8a8f6194630b2/bin/esbuild b/.next/node_modules/esbuild-caa8a8f6194630b2/bin/esbuild
old mode 100644
new mode 100755
index c0f429954864b4cedcba70d5ca2a5123efdaa7e4..7ec636c39b0b417acf1d15e3899893254776ffba
--- a/.next/node_modules/esbuild-caa8a8f6194630b2/bin/esbuild
+++ b/.next/node_modules/esbuild-caa8a8f6194630b2/bin/esbuild
@@ -201,7 +201,7 @@ for your current platform.`);
         "node_modules",
         ".cache",
         "esbuild",
-        `pnpapi-${pkg.replace("/", "-")}-${"0.25.10"}-${path.basename(subpath)}`
+        `pnpapi-${pkg.replace("/", "-")}-${"0.27.3"}-${path.basename(subpath)}`
       );
       if (!fs.existsSync(binTargetPath)) {
         fs.mkdirSync(path.dirname(binTargetPath), { recursive: true });
diff --git a/.next/node_modules/esbuild-caa8a8f6194630b2/lib/main.js b/.next/node_modules/esbuild-caa8a8f6194630b2/lib/main.js
index 0f61c81621ded9262d532307857e252673c76473..1e0cafb750b8b8a972edd2ee3fb9a31c2ac10a3e 100644
--- a/.next/node_modules/esbuild-caa8a8f6194630b2/lib/main.js
+++ b/.next/node_modules/esbuild-caa8a8f6194630b2/lib/main.js
@@ -643,8 +643,8 @@ function createChannel(streamIn) {
     if (isFirstPacket) {
       isFirstPacket = false;
       let binaryVersion = String.fromCharCode(...bytes);
-      if (binaryVersion !== "0.25.10") {
-        throw new Error(`Cannot start service: Host version "${"0.25.10"}" does not match binary version ${quote(binaryVersion)}`);
+      if (binaryVersion !== "0.27.3") {
+        throw new Error(`Cannot start service: Host version "${"0.27.3"}" does not match binary version ${quote(binaryVersion)}`);
       }
       return;
     }
@@ -1770,7 +1770,7 @@ for your current platform.`);
         "node_modules",
         ".cache",
         "esbuild",
-        `pnpapi-${pkg.replace("/", "-")}-${"0.25.10"}-${path.basename(subpath)}`
+        `pnpapi-${pkg.replace("/", "-")}-${"0.27.3"}-${path.basename(subpath)}`
       );
       if (!fs.existsSync(binTargetPath)) {
         fs.mkdirSync(path.dirname(binTargetPath), { recursive: true });
@@ -1805,7 +1805,7 @@ if (process.env.ESBUILD_WORKER_THREADS !== "0") {
   }
 }
 var _a;
-var isInternalWorkerThread = ((_a = worker_threads == null ? void 0 : worker_threads.workerData) == null ? void 0 : _a.esbuildVersion) === "0.25.10";
+var isInternalWorkerThread = ((_a = worker_threads == null ? void 0 : worker_threads.workerData) == null ? void 0 : _a.esbuildVersion) === "0.27.3";
 var esbuildCommandAndArgs = () => {
   if ((!ESBUILD_BINARY_PATH || false) && (path2.basename(__filename) !== "main.js" || path2.basename(__dirname) !== "lib")) {
     throw new Error(
@@ -1872,7 +1872,7 @@ var fsAsync = {
     }
   }
 };
-var version = "0.25.10";
+var version = "0.27.3";
 var build = (options) => ensureServiceIsRunning().build(options);
 var context = (buildOptions) => ensureServiceIsRunning().context(buildOptions);
 var transform = (input, options) => ensureServiceIsRunning().transform(input, options);
@@ -1975,7 +1975,7 @@ var stopService;
 var ensureServiceIsRunning = () => {
   if (longLivedService) return longLivedService;
   let [command, args] = esbuildCommandAndArgs();
-  let child = child_process.spawn(command, args.concat(`--service=${"0.25.10"}`, "--ping"), {
+  let child = child_process.spawn(command, args.concat(`--service=${"0.27.3"}`, "--ping"), {
     windowsHide: true,
     stdio: ["pipe", "pipe", "inherit"],
     cwd: defaultWD
@@ -2079,7 +2079,7 @@ var runServiceSync = (callback) => {
     esbuild: node_exports
   });
   callback(service);
-  let stdout = child_process.execFileSync(command, args.concat(`--service=${"0.25.10"}`), {
+  let stdout = child_process.execFileSync(command, args.concat(`--service=${"0.27.3"}`), {
     cwd: defaultWD,
     windowsHide: true,
     input: stdin,
@@ -2099,7 +2099,7 @@ var workerThreadService = null;
 var startWorkerThreadService = (worker_threads2) => {
   let { port1: mainPort, port2: workerPort } = new worker_threads2.MessageChannel();
   let worker = new worker_threads2.Worker(__filename, {
-    workerData: { workerPort, defaultWD, esbuildVersion: "0.25.10" },
+    workerData: { workerPort, defaultWD, esbuildVersion: "0.27.3" },
     transferList: [workerPort],
     // From node's documentation: https://nodejs.org/api/worker_threads.html
     //
diff --git a/.next/node_modules/esbuild-caa8a8f6194630b2/node_modules/.bin/esbuild b/.next/node_modules/esbuild-caa8a8f6194630b2/node_modules/.bin/esbuild
new file mode 100755
index 0000000000000000000000000000000000000000..906754c12b4081b2f522f4f9e8b84d871418c101
--- /dev/null
+++ b/.next/node_modules/esbuild-caa8a8f6194630b2/node_modules/.bin/esbuild
@@ -0,0 +1,16 @@
+#!/bin/sh
+basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
+
+case `uname` in
+    *CYGWIN*|*MINGW*|*MSYS*)
+        if command -v cygpath > /dev/null 2>&1; then
+            basedir=`cygpath -w "$basedir"`
+        fi
+    ;;
+esac
+
+if [ -x "$basedir/node" ]; then
+  exec "$basedir/node"  "$basedir/../../bin/esbuild" "$@"
+else
+  exec node  "$basedir/../../bin/esbuild" "$@"
+fi
diff --git a/.next/node_modules/esbuild-caa8a8f6194630b2/package.json b/.next/node_modules/esbuild-caa8a8f6194630b2/package.json
index 028ea2dc62549995f8fe12fc94608d5c85b42ed2..0b82fd8b5a5a99f56b44a7fa9e0dc293c7fc78b3 100644
--- a/.next/node_modules/esbuild-caa8a8f6194630b2/package.json
+++ b/.next/node_modules/esbuild-caa8a8f6194630b2/package.json
@@ -1,6 +1,6 @@
 {
   "name": "esbuild",
-  "version": "0.25.10",
+  "version": "0.27.3",
   "description": "An extremely fast JavaScript and CSS bundler and minifier.",
   "repository": {
     "type": "git",
@@ -18,32 +18,32 @@
     "esbuild": "bin/esbuild"
   },
   "optionalDependencies": {
-    "@esbuild/aix-ppc64": "0.25.10",
-    "@esbuild/android-arm": "0.25.10",
-    "@esbuild/android-arm64": "0.25.10",
-    "@esbuild/android-x64": "0.25.10",
-    "@esbuild/darwin-arm64": "0.25.10",
-    "@esbuild/darwin-x64": "0.25.10",
-    "@esbuild/freebsd-arm64": "0.25.10",
-    "@esbuild/freebsd-x64": "0.25.10",
-    "@esbuild/linux-arm": "0.25.10",
-    "@esbuild/linux-arm64": "0.25.10",
-    "@esbuild/linux-ia32": "0.25.10",
-    "@esbuild/linux-loong64": "0.25.10",
-    "@esbuild/linux-mips64el": "0.25.10",
-    "@esbuild/linux-ppc64": "0.25.10",
-    "@esbuild/linux-riscv64": "0.25.10",
-    "@esbuild/linux-s390x": "0.25.10",
-    "@esbuild/linux-x64": "0.25.10",
-    "@esbuild/netbsd-arm64": "0.25.10",
-    "@esbuild/netbsd-x64": "0.25.10",
-    "@esbuild/openbsd-arm64": "0.25.10",
-    "@esbuild/openbsd-x64": "0.25.10",
-    "@esbuild/openharmony-arm64": "0.25.10",
-    "@esbuild/sunos-x64": "0.25.10",
-    "@esbuild/win32-arm64": "0.25.10",
-    "@esbuild/win32-ia32": "0.25.10",
-    "@esbuild/win32-x64": "0.25.10"
+    "@esbuild/aix-ppc64": "0.27.3",
+    "@esbuild/android-arm": "0.27.3",
+    "@esbuild/android-arm64": "0.27.3",
+    "@esbuild/android-x64": "0.27.3",
+    "@esbuild/darwin-arm64": "0.27.3",
+    "@esbuild/darwin-x64": "0.27.3",
+    "@esbuild/freebsd-arm64": "0.27.3",
+    "@esbuild/freebsd-x64": "0.27.3",
+    "@esbuild/linux-arm": "0.27.3",
+    "@esbuild/linux-arm64": "0.27.3",
+    "@esbuild/linux-ia32": "0.27.3",
+    "@esbuild/linux-loong64": "0.27.3",
+    "@esbuild/linux-mips64el": "0.27.3",
+    "@esbuild/linux-ppc64": "0.27.3",
+    "@esbuild/linux-riscv64": "0.27.3",
+    "@esbuild/linux-s390x": "0.27.3",
+    "@esbuild/linux-x64": "0.27.3",
+    "@esbuild/netbsd-arm64": "0.27.3",
+    "@esbuild/netbsd-x64": "0.27.3",
+    "@esbuild/openbsd-arm64": "0.27.3",
+    "@esbuild/openbsd-x64": "0.27.3",
+    "@esbuild/openharmony-arm64": "0.27.3",
+    "@esbuild/sunos-x64": "0.27.3",
+    "@esbuild/win32-arm64": "0.27.3",
+    "@esbuild/win32-ia32": "0.27.3",
+    "@esbuild/win32-x64": "0.27.3"
   },
   "license": "MIT"
 }
diff --git a/package.json b/package.json
index 47d07718d5bd0d135efa3fdcb6d5ae41c1bd92b8..0bbe509b3318d3037ee5fca8c257c8cc5237f452 100644
--- a/package.json
+++ b/package.json
@@ -17,7 +17,7 @@
     ".next/**/*"
   ],
   "dependencies": {
-    "esbuild": "0.25.10",
+    "esbuild": "0.27.3",
     "next": "16.1.6"
   },
   "devDependencies": {
